packages/matter.js/src/codec/BtpCodec.ts

```diff
@@ -0,0 +1,66 @@
+/**
+ * @license
+ * Copyright 2022-2023 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { ByteArray, Endian } from "../util/ByteArray.js";
+
+export interface BTPHandshakeRequest {
+    sourceBits: SourceBits,
+    managementOpcode?: string,
```

Comment:
Why string? it is a number

Reply:
managementOpcode = 6c (from the example - 656c04000000b90006), so shouldn't that be a string?

Reply:
In fact it is 0x6c as number. So I would create an enum with the defined ManagwmentOpcodes and give them a name (and right now only 0x6c is defiend)

Reply:
Done!